lusidtools/cocoon/seed_sample_data.py

```diff
@@ -0,0 +1,49 @@
+import pandas as pd
+from lusidtools.cocoon.cocoon import load_from_data_frame
+from lusidtools.cocoon.utilities import load_json_file
+import logging
+
+logger = logging.getLogger()
+
+mappings = dict(load_json_file("config/seed_sample_data.json"))
+
+def seed_sample_data(api_factory, file_types, scope: str, transaction_source, mappings: dict = mappings):
```

Comment:
could probably just call this `seed_data`

Reply:
Sure. Done.